006-connected-udp-sockets.md

```diff
@@ -0,0 +1,67 @@
+| Title  | Connected UDP sockets |
+|--------|-----------------------|
+| Author | @santigimeno          |
+| Status | DRAFT                 |
+| Date   | 2017-03-25 21:41:00   |
+
+
+## Overview
+
+Calling `connect()` on a UDP socket associates a remote address and port to the
+socket, so every message sent by this socket is automatically sent to that
+destination.
+
+## API
+
+The following functions are defined:
+
+```c
+int uv_udp_connect(uv_udp_t* handle, const struct sockaddr* addr);
+
+int uv_udp_disconnect(uv_udp_t* handle);
```

Comment:
> Connectionless  sockets  may dissolve the association by connecting to an address with the sa_family member of sockaddr set to AF_UNSPEC (supported on Linux  since  kernel 2.2).